### PR TITLE
Improvement/version upgrade

### PR DESCRIPTION
**Pull request by theshohidul**

### CODE REVIEW

File: app/.gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 __pycache__
```
Comment (sazzad-sazib) [inline app/.gitignore:1]: pycache roye gese

File: docker/conf/python/Dockerfile [hunk 2]
```diff
@@ -1,20 +1,24 @@
-FROM python:3.9
```
Comment (sazzad-sazib) [inline docker/conf/python/Dockerfile:1]: docker/.DS_Store roye gese

Comment (theshohidul) [reply docker/conf/python/Dockerfile:1]: Thanks, will remove those.

File: app/.gitignore (hunk 1 above)
Comment (theshohidul) [reply app/.gitignore:1]: Thanks, will remove those.

Comment (theshohidul) [reply app/.gitignore:1]: Removed and added in gitignore

File: docker/conf/python/Dockerfile (hunk 2 above)
Comment (theshohidul) [reply docker/conf/python/Dockerfile:1]: Removed and added in gitignore

**Pull request merged**